metric-publishers/emf-metric-publisher/pom.xml

```diff
@@ -71,6 +67,18 @@
             <version>${awsjavasdk.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.networknt</groupId>
+            <artifactId>json-schema-validator</artifactId>
+            <version>1.4.0</version>
```

Comment:
Looks like 1.5.4 is the latest version https://github.com/networknt/json-schema-validator/blob/master/pom.xml#L26C11-L26C17 

We should define all versions in the top pom.xml to make it easier to manage, see example: https://github.com/aws/aws-sdk-java-v2/blob/master/pom.xml#L146